cpack/PackageBundle.cmake

```diff
@@ -0,0 +1,18 @@
+# Heavily based on https://github.com/OpenTTD/OpenTTD/blob/8d54f765392654ab634ba3a950c56dd0bf1e7dd9/cmake/PackageBundle.cmake
+
+set(CPACK_BUNDLE_NAME "FreeRCT")
+set(CPACK_BUNDLE_ICON "${CMAKE_SOURCE_DIR}/graphics/sprites/logo/logo.png")
+set(CPACK_BUNDLE_PLIST "${CMAKE_CURRENT_BINARY_DIR}/Info.plist.in")
+# https://cmake.org/cmake/help/latest/cpack_gen/bundle.html#variable:CPACK_BUNDLE_STARTUP_COMMAND
+set(CPACK_BUNDLE_STARTUP_COMMAND "${CMAKE_CURRENT_BINARY_DIR}/bin/freerct")
+set(CPACK_DMG_FORMAT "UDBZ")
+
+configure_file("${CMAKE_SOURCE_DIR}/packaging_data/osx/Info.plist.in" "${CMAKE_CURRENT_BINARY_DIR}/Info.plist.in")
+set(CPACK_BUNDLE_PLIST_SOURCE "${CMAKE_CURRENT_BINARY_DIR}/Info.plist.in")
+
+# Link libs, the wrong way (libpng sdl2 sdl2_ttf)
```

Comment:
From this bit onwards the things done in this file are still completely broken, libraries are in fact not linked and the builds still don't work without manually installing them with homebrew.